Dockerfile

```diff
@@ -1,20 +1,24 @@
-FROM aquasec/trivy:0.32.1
+FROM aquasec/trivy:0.48.3
 
 RUN apk add --no-cache --upgrade bash
 RUN apk add jq
 
 COPY build.sh .
 COPY imageTrivyScanner.sh .
 COPY filesystemTrivyScanner.sh .
+COPY repoTrivyScanner.sh .
+
 COPY BP-BASE-SHELL-STEPS/functions.sh .
 COPY BP-BASE-SHELL-STEPS/log-functions.sh .
 
+ENV IMAGE_NAME registry.buildpiper.in/trivy-scan
```

Comment:
Set empty env variable is it it's a mandatory imput.
Also this MR should only bhe file system licence scanning so don't have it at all